src/documentation/config/config.area.preprocess.md

```diff
@@ -0,0 +1,42 @@
+#### **preprocess** : {<%= type %>}
+
+*default: function minMaxFilter(1000)*
+
+Sometimes, usually for performance reasons, you don't want Contour to draw *all* of the points in your data set. 
+
+The default value is a function that:
+
+* Takes in an integer, which is 1000 in the default case.
+* If the number of points in your data set is less than or equal to this, simply returns the entire data set.
+* Otherwise, returns a filtered data set. The filtering creates a new, smaller, data set, which has at most two data points for each x value in the original data: one with the minimum y value and one with the maximum y value.
+
```

Comment:
this should be something like "points for each _screen_ x coordinate in the original..." this means, you could have different x values, but if they will project onto the same x screen coordinate, is when they'd get filtered